js/views/pauseScreen.js

```diff
@@ -104,41 +108,20 @@ class PauseScreen extends React.Component {
     this.props.handleVrPlayerClick();
   }
 
-  /**
-   * call handleTouchEnd when touchend was called on selectedScreen and videoType is Vr
-   * @param {Event} e - event object
-   */
-  handleTouchEnd(e) {
-    if (this.props.controller.state.controlBarVisible) {
-      var shouldToggle = false;
-      if (this.props.controller.videoVr) {
-        e.preventDefault();
-        if (!this.props.isVrMouseMove) {
-          shouldToggle = true;
-        }
-      } else {
-        shouldToggle = true;
-      }
-      if (shouldToggle) {
-        this.props.controller.togglePlayPause(e);
-      }
-    }
-  }
-
   /**
    * call handleVrTouchEnd when touchend was called on selectedScreen and videoType is Vr
-   * @param {Event} e - event object
+   * @param {Event} event - event object
    */
-  handleVrTouchEnd(e) {
-    this.props.handleVrPlayerMouseUp(e);
+  handleVrTouchEnd(event) {
+    this.props.handleVrPlayerMouseUp(event);
   }
 
   /**
    * remove the button on pause screen for correct checking mouse movement
    */
   hideVrPauseButton() {
```

Comment:
This method is a kind overcomplicated, and it runs setTimeout even if not required. Guess it should be:
if (!this.props.controller.videoVr || !this.pauseButton){
  return;
}
setTimeout(() => this.pauseButton.style.display = 'none', 1000);

Reply:
\+

Reply:
\+